Enhance zh-cn/learn/html/index.md. (#5817)
mdn/translated-content: files/zh-cn/learn/html/index.md

---
title: 学习 HTML ：指南与教程

> slug: learn/HTML

Reviewer: ```suggestion
slug: Learn/HTML
```

tags:
  - HTML
  - HTML5

>   - 章节

Reviewer: 在英文文档中，使用了 `topic` 一词，而且“章节”包含了“章”和“节”两部分，节的概念是很小的，一般由几个自然段组成。将主题换为章节可能不是一个合适的做法。

  - 入门指导
  - 基础
  - 学习
  - 新手
  - 简介
translation_of: Learn/HTML
---

{{LearnSidebar}}

为了创建一个网站，你需要了解 {{Glossary('HTML')}} —— 一项用于定义网页结构的基本技术。HTML 用于标识你的网页内容是应该被解析为段落、列表、头部、链接、图像、多媒体播放器、表单或是其他众多可用的元素之一，亦或是你定义的新元素。

> **标注：**
>
> #### 想要成为一名前端工程师？
>
> 我们汇总了一门课程，其中包含你实现成为前端工程师目标所需要的所有基本信息。
>

> > [**点击开始**](/zh-CN/docs/Learn/Front-end_web_developer)

Reviewer: 这样是否更好：

```suggestion
> [**从这儿开始**](/zh-CN/docs/Learn/Front-end_web_developer)
```

Author: 此处来源是 files/zh-cn/learn/index.md#L32 修复此处后我稍后再开新 PR 处理那里的问题

Reviewer: 如果仅有这个小问题吗，可以暂不另开PR修复

## 须知

在开始这个章节的学习之前，你至少要基本熟悉使用电脑和被动地使用网络（即单纯地查看内容）。你应该设置好一个基础的工作环境，详细参照 [安装基础软件](/zh-CN/docs/Learn/Getting_started_with_the_web/Installing_basic_software), 并且理解如何新建和管理文件，详细参照[文件处理](/zh-CN/docs/Learn/Getting_started_with_the_web/Dealing_with_files)——这两者都在 [Web 入门](/zh-CN/docs/Learn/Getting_started_with_the_web) 的完全新手模块里。

> 在尝试学习这个章节之前，建议先完成 [Web 入门](/zh-CN/docs/Learn/Getting_started_with_the_web)章节，不过这并不是必要的。[HTML 基础](/zh-CN/docs/Learn/Getting_started_with_the_web/HTML_basics)里大多数的文章在 [HTML 介绍](/zh-CN/docs/Learn/HTML/Introduction_to_HTML)模块中也有涉及，不过要详细得多。

Reviewer: 除了上面提出的“章节”一词的准确性问题外，我建议作如下修改，你看这样是否更加合适

```suggestion
在尝试学习这个章节之前，建议先完成 [Web 入门](/zh-CN/docs/Learn/Getting_started_with_the_web)，不过这并不是必要的。[HTML 基础](/zh-CN/docs/Learn/Getting_started_with_the_web/HTML_basics)里大多数的文章在 [HTML 介绍](/zh-CN/docs/Learn/HTML/Introduction_to_HTML)中也有涉及，不过要详细得多。
```

在学习了 HTML 之后，你就可以继续学习其他进阶章节了，例如：

- [CSS](/zh-CN/docs/Learn/CSS)，以及如何用它装饰 HTML (例如：更改你的文本字号和字体，添加边框和阴影，将你的页面设计成多栏布局，添加动画和其他视觉效果。)
- [JavaScript](/zh-CN/docs/Learn/JavaScript)，以及如何用它为网页添加动态功能(例如：找到并在地图上绘制出你的地址，触发按钮时让 UI 元素显示或消失，将用户的数据本地储存在他们的电脑里等等。))

## 模块

这个章节由包含以下模块，建议从第一个开始，按顺序进行学习。

- [HTML 介绍](/zh-CN/docs/Learn/HTML/Introduction_to_HTML)
  - : 这一模块将为你铺路，帮你习惯一些重要的概念和语法，着眼于如何对文本应用 HTML，创造超链接，以及使用 HTML 构造一个网页。
- [多媒体和嵌入](/zh-CN/docs/Learn/HTML/Multimedia_and_embedding)
  - : 这个模块带你探索如何使用 HTML 在你的网页里如何包含多媒体信息，例如用各种方法包含图片，以及嵌入视频、音频，甚至是嵌入其他完整的网页。
- [HTML 表格](/zh-CN/docs/Learn/HTML/Tables)
  - : 在网页上用易于理解和可访问{{glossary("Accessibility", "accessible")}} 的方式来表示表格数据是一项挑战。这个模块包括了基本的表格标记及更多复杂的特性，例如实现标题和总结。

## 解决常见的 HTML 问题

[使用 HTML 解决常见问题](/zh-CN/docs/Learn/HTML/Howto)提供了一系列在创建网页过程中可能遇到的常见问题的解决方案：处理标题，添加图片或视频，强调内容，创建一个基础表单等等。

> ## 相关链接

Reviewer: 请协助更新 `See also` 的翻译

```suggestion
## 参见
```

  - 简介
translation_of: Learn/HTML
---

> 

Reviewer: 这儿不需要空行：
```suggestion
```

## 须知

在开始这个主题的学习之前，你至少要基本熟悉使用电脑和被动地使用网络（即单纯地查看内容）。你应该设置好一个基础的工作环境，详细参照 [安装基础软件](/zh-CN/docs/Learn/Getting_started_with_the_web/Installing_basic_software), 并且理解如何新建和管理文件，详细参照[文件处理](/zh-CN/docs/Learn/Getting_started_with_the_web/Dealing_with_files)——这两者都在 [Web 入门](/zh-CN/docs/Learn/Getting_started_with_the_web) 的完全新手模块里。

在尝试学习这个主题之前，建议先完成 [Web 入门](/zh-CN/docs/Learn/Getting_started_with_the_web)主题，不过这并不是必要的。[HTML 基础](/zh-CN/docs/Learn/Getting_started_with_the_web/HTML_basics)里大多数的文章在 [HTML 介绍](/zh-CN/docs/Learn/HTML/Introduction_to_HTML)中也有涉及，不过要详细得多。

在学习了 HTML 之后，你就可以继续学习其他进阶主题了，例如：

- [CSS](/zh-CN/docs/Learn/CSS)，以及如何用它装饰 HTML (例如：更改你的文本字号和字体，添加边框和阴影，将你的页面设计成多栏布局，添加动画和其他视觉效果。)
- [JavaScript](/zh-CN/docs/Learn/JavaScript)，以及如何用它为网页添加动态功能(例如：找到并在地图上绘制出你的地址，触发按钮时让 UI 元素显示或消失，将用户的数据本地储存在他们的电脑里等等。))

## 模块

这个主题由包含以下模块，建议从第一个开始，按顺序进行学习。

- [HTML 介绍](/zh-CN/docs/Learn/HTML/Introduction_to_HTML)
  - : 这一模块将为你铺路，帮你习惯一些重要的概念和语法，着眼于如何对文本应用 HTML，创造超链接，以及使用 HTML 构造一个网页。
- [多媒体和嵌入](/zh-CN/docs/Learn/HTML/Multimedia_and_embedding)
  - : 这个模块带你探索如何使用 HTML 在你的网页里如何包含多媒体信息，例如用各种方法包含图片，以及嵌入视频、音频，甚至是嵌入其他完整的网页。
- [HTML 表格](/zh-CN/docs/Learn/HTML/Tables)

>   - : 在网页上用易于理解和可访问{{glossary("Accessibility", "accessible")}} 的方式来表示表格数据是一项挑战。这个模块包括了基本的表格标记及更多复杂的特性，例如实现标题和总结。

Reviewer: ```suggestion
  - : 在网页上用易于理解和{{glossary("Accessibility", "无障碍")}}的方式来表示表格数据是一项挑战。这个模块包括了基本的表格标记及更多复杂的特性，例如实现标题和总结。
```
